Description: Don't traverse into autopilot, not available on Debian.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: not needed, Debian-specific

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -1,2 +1 @@
-add_subdirectory(autopilot)
 add_subdirectory(unittests)
--- a/tests/unittests/CMakeLists.txt
+++ b/tests/unittests/CMakeLists.txt
@@ -33,7 +33,6 @@
 
 qt5_use_modules(tst_storagemonitor Widgets Core Quick Qml Test)
 add_test(tst_storagemonitor tst_storagemonitor -xunitxml -o test.xml)
-add_test(flake8 python3 -m flake8 ${CMAKE_SOURCE_DIR}/tests/autopilot)
 set_tests_properties(tst_storagemonitor PROPERTIES
     TIMEOUT ${CTEST_TESTING_TIMEOUT}
     ENVIRONMENT "QT_QPA_PLATFORM=minimal"
